app/src/main/java/com/dessalines/thumbkey/ui/components/settings/behavior/BehaviorActivity.kt

```diff
@@ -381,6 +385,22 @@ fun BehaviorActivity(
                             )
                         },
                     )
+                    SwitchPreference(
+                        value = ghostKeysEnabledState,
+                        onValueChange = {
+                            ghostKeysEnabledState = it
+                            updateBehavior()
+                        },
+                        title = {
+                            Text(stringResource(R.string.ghost_keys_enable))
+                        },
```

Comment:
You can add a `summary = { Text(R.string.ghost_keys_description` , to elaborate on what they are. It'd also be good to add them to the README in the user guide section.

Reply:
Sure. I've written a description and added a part to the README. Hopefully it's clear, but let me know if you prefer I change something